files/SQLite.props

```diff
@@ -1,6 +1,6 @@
 <?xml version="1.0" encoding="utf-8" ?>
 <Project ToolsVersion="12.0" xmlns="http://schemas.microsoft.com/developer/msbuild/2003">
-  <ItemGroup Condition="!Exists('$(MSBuildProjectName).project.json') And !Exists('project.json') And '$(Platform)' == 'AnyCPU'">
+  <ItemGroup Condition="!Exists('$(MSBuildProjectName).project.json') And !Exists('project.json') And '$(Platform)' == 'AnyCPU' And '$(ExcludeSqliteNative)' != 'true' ">
```

Comment:
The msbuild props in this file are meant only for NuGet2 projects. In the (hopefully unlikely) event that project.json is renamed or moved to a different file, we still need a way to stop this NuGet2 target and overlapping with what NuGet3 already does automagically. 

If there is a better way to express `if nugetClientVersion == 2` in msbuild, then I'm all for using that instead.

cc @emgarten